yt_dlp/postprocessor/embedthumbnail.py

```diff
@@ -90,12 +90,33 @@ def run(self, info):
 
         success = True
         if info['ext'] == 'mp3':
-            options = [
-                '-c', 'copy', '-map', '0:0', '-map', '1:0', '-write_id3v1', '1', '-id3v2_version', '3',
-                '-metadata:s:v', 'title="Album cover"', '-metadata:s:v', 'comment=Cover (front)']
+            # Method 1: Use mutagen
+            # Prioritize mutagen over ffmpeg since ffmpeg messes up the lyrics data
+            if mutagen:
+                try:
+                    self._report_run('mutagen', filename)
+                    audio = mutagen.id3.ID3(filename)
+                    if 'APIC' in audio:
+                        del audio['APIC']
```

Comment:
Why do we need that? Since we use `audio['APIC'] = ...`, this shouldnt be required?
```suggestion
```